scripts/runSimpleDatasafeAdapterPerformanceTest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+if [ $# -eq 0 ]; then
```

Comment:
I think it is enough to print which environment variables script expects and if some required are not set, script should abort. It is unsafe to enter them into command line. Something like 
```
if [ -z "$AWS_ACCESS_KEY" ] ; then
echo 'Missing AWS_ACCESS_KEY environment variable'
help()
exit 1
fi
```
for each required variable
and welcome screen in help() function that prints all variables

Reply:
ok